### PR TITLE
Add extra param in Adyen settings for DCC

### PR DESCRIPTION
**Pull request by dara123**
## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

File: openapi/components/schemas/GatewayAccountConfig/Adyen.yaml [hunk 1]
```diff
@@ -46,6 +46,15 @@ allOf:
...
                   example: 0.50
+                amount:
+                  description: Fix amount.
+                  type: number
+                  format: integer
+                  example: 100
+                currency:
+                  description: Three letter currency code.
+                  type: string
+                  example: USD
```
Comment (ashkarpetin) [inline openapi/components/schemas/GatewayAccountConfig/Adyen.yaml:57]:
```suggestion
                currency:
                  $ref: ../CurrencyCode.yaml
```

File: openapi/components/schemas/GatewayAccountConfig/Adyen.yaml [hunk 2]
```diff
@@ -46,6 +46,13 @@ allOf:
                   minimum: 0.00
                   maximum: 100.00
                   example: 0.50
+                amount:
+                  description: Can be set for fix commission amount in minor unit.
```
Comment (noon-dawg) [inline openapi/components/schemas/GatewayAccountConfig/Adyen.yaml:50]: suggestion:
```suggestion
                  description: Fixed commission amount in the minor unit.
```

**Review by noon-dawg — DISMISSED** [review]
Looks good on the docs side

File: openapi/components/schemas/GatewayAccountConfig/Adyen.yaml [hunk 3]
```diff
@@ -46,6 +46,13 @@ allOf:
                   minimum: 0.00
                   maximum: 100.00
                   example: 0.50
+                amount:
+                  description: Fixed commission amount in the minor unit.
```
Comment (ashkarpetin) [inline openapi/components/schemas/GatewayAccountConfig/Adyen.yaml:50]:
```suggestion
                  description: Fixed commission amount in minor units.
```

**Comment by dara123** [comment]
@Rebilly/php please review

**Comment by dara123** [comment]
@Rebilly/php please review

**Review by colinbird — APPROVED** [review]

**Review by lexerom — APPROVED** [review]

**Pull request merged**